src/components/Organisms/EmailSignUp/EmailSignUp.js

```diff
@@ -75,7 +75,7 @@ const EmailSignUp = ({
             value={value}
             onChange={event => setValue(event.target.value)}
           />
-          <ButtonWrapper backgroundColor={backgroundColor}>
+          <ButtonWrapper>
```

Comment:
Why don't we need this any more?

As an aside, we should probably be making these kind of adjustments in a separate PR, as it kinda muddies the water for a PR that's supposed to just cover module upgrades.

Reply:
No need of background colour it is wrong just a clean up.